Owncloud iOs Client/DataBase/Queries/ManageDB.m

```diff
@@ -992,4 +993,95 @@ + (void) updateDBVersion16To17{
     
 }
 
+///-----------------------------------
+/// @name Update Database version with 17 version to 18
+///-----------------------------------
+
+/**
+ * Changes:
+ *
+ * Alter users table, adds new field to track user background Instant Upload preference
+ */
++ (void) updateDBVersion17To18 {
+    
+    BOOL defaultBackgroundInstantUploadValue = (BOOL)[ManageAppSettingsDB isInstantUpload]; //Users who are using Instant Upload before this migration also have background upload enabled, so we want to default to enabling this preference.
+    
+    FMDatabaseQueue *queue = Managers.sharedDatabase;
+    
+    // Get the old instant upload date so it can be migrated to an NSTimestamp
+    __block long lastDateInstantUpload;
+    [queue inDatabase:^(FMDatabase *db) {
+        FMResultSet *rs = [db executeQuery:@"SELECT date_instant_upload FROM users WHERE activeaccount=1"];
```

Comment:
I mean, use this query adding the column instant_upload
```
 [db executeQuery:@"SELECT date_instant_upload, instant_upload FROM users WHERE activeaccount=1"];
```

to avoid this line:
```
  BOOL defaultBackgroundInstantUploadValue = (BOOL)[ManageAppSettingsDB isInstantUpload];
```
that is doing another query to the same table.